packages/lime-librenet6/files/etc/uci-defaults/tinc-setup

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+# tell tinc to use the librenet6 network
+uci -q show tinc.librenet6 || {
+    touch /etc/config/tinc
+    uci batch <<-EOF
+    set tinc.librenet6='tinc-net'
+    set tinc.librenet6.enabled='1'
+    commit tinc
+EOF
+}
+
+# disable ra
+uci set network.lan.accept_ra=0
+uci commit network
+
+# set IPv6 subnet of librenet6.librenet.org
+uci set lime.network.main_ipv6_address '2002:8aff:5905:%N1%N2::/64'
```

Comment:
This seems to interfere with normal lime behavior or am I missing something?

Reply:
Well you have to use an IP of that subnet if you want a working connection. To keep it simple for users I decided to add this